+MonteCarlo/@ParticleWalker/private/getLimitedStep.m

```diff
@@ -5,7 +5,7 @@
 
 % step in space
 maxStep_squared = maxStepLength^2; % pre-compute here
-dxdydz = zeros(1, dim);
+dxdydz = zeros(1, 3);
```

Comment:
add `mask = contains({'x', 'y', 'z'}, cellstr(dim.'));` to compute the [x, y, z] mask from `'xyz'` string `dim`